Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerMentionCellView_Tests.swift

```diff
@@ -0,0 +1,134 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatMessageComposerMentionCellView_Tests: XCTestCase {
+    /// Default reference width for the cell. Not related to any screen size.
+    private static var defaultCellWidth: CGFloat = 300
+    /// To simulate cell behavior with autoresizing height according to content
+    /// we create this height value as well.
+    private static var defaultCellHeight: CGFloat = 50
+
+    func test_emptyAppearance() {
+        let view = ChatMessageComposerMentionCellView().withoutAutoresizingMaskConstraints
+        view.widthAnchor.constraint(equalToConstant: Self.defaultCellWidth).isActive = true
+        view.heightAnchor.constraint(greaterThanOrEqualToConstant: Self.defaultCellHeight).isActive = true
+
+        AssertSnapshot(view, variants: .onlyUserInterfaceStyles)
```

Comment:
This should be just `AssertSnapshot(view)`, you want to test different content sizes, not just light/dark mode.